docs/src/man/guide.md

```diff
@@ -2,134 +2,123 @@
 
 Documenter is designed to do one thing -- combine markdown files and inline docstrings from
 Julia's docsystem into a single inter-linked document. What follows is a step-by-step guide
-to creating a simple document.
+intended for users who iteratively develop a package and write documentation for it.
 
+## Preliminaries
 
```

Comment:
I would add a paragraph here about why you'd want to use this more interactive workflow. E.g. something along these lines (but edits definitely welcome):

```suggestion

While the standard Documenter setup allows package documentation to be built on the command line by simply executing the `make.jl` script with e.g. `julia --project=docs/ docs/make.jl`, this is can be slow due to all the re-compilation happening in every new Julia session.

```